docker-compose.yaml

```diff
@@ -1,9 +1,28 @@
 version: '3'
+
 services:
+
+
+  adminer:
+    image: adminer
```

Comment:
PhpMyAdmin egy fokkal szofisztikáltabb erre a célra (például tud importot is meg sok egyéb dolgot).